<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuWidget</class>
 <widget class="QWidget" name="MainMenuWidget">
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>

  <layout class="QVBoxLayout" name="central_layout">
   <!-- Top spacer -->
   <item>
    <spacer name="top_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
      </size>
     </property>
    </spacer>
   </item>

   <!-- Logo -->
   <item>
    <widget class="QLabel" name="logo_label">
     <property name="text">
      <string>TRON</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>

   <item>
    <spacer name="between_logo_and_buttons">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
      </size>
     </property>
    </spacer>
   </item>

   <!-- Buttons -->
   <item>
    <layout class="QVBoxLayout" name="main_screen_layout">
     <property name="alignment">
      <set>Qt::AlignHCenter</set>
     </property>
     <property name="spacing">
      <number>15</number>
     </property>

     <item>
      <widget class="QPushButton" name="start_button">
       <property name="text">
        <string>start</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="settings_button">
       <property name="text">
        <string>settings</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="creators_info_button">
       <property name="text">
        <string>creators</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="quit_button">
       <property name="text">
        <string>quit</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <!-- Bottom spacer -->
   <item>
    <spacer name="bottom_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>